py4DSTEM/process/calibration/origin.py

```diff
@@ -514,6 +434,12 @@ def get_origin_beamstop_braggpeaks(braggpeaks,center_guess,radii,Q_Nx,Q_Ny,
     return qx0,qy0
 
 
+def get_origin_brightest_disk(
+        datacube,
+        **kwargs
+        ):
+        raise Exception("This function needs to be added. Try another origin detection mode.")
```

Comment:
This method was still in the facade function's documentation, but the function itself was gone and it was not in the valid `mode` args or in the function dictionary.  I don't have a strong preference about removing the function vs. adding it back in - this was not my favorite approach because it runs an entire bout of disk detection and stores nothing but the center disk, but if it's useful to some folks it can definitely stay. Either way, I want to make sure we make the code and docstrings consistent.  Noting this here so folks can weigh in if they have preferences either way.